Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -97,19 +97,16 @@ EmbeddedFrag::EmbeddedFrag(const RDKit::ROMol *mol,
   // now for points that new atoms will be added to later on we need to do some
   // setup
   for (auto dai : d_attachPts) {
-    RDKit::ROMol::ADJ_ITER nbrIdx, endNbrs;
-    boost::tie(nbrIdx, endNbrs) =
-        dp_mol->getAtomNeighbors(dp_mol->getAtomWithIdx(dai));
     // find the neighbors that are already embedded for each of these atoms
     RDKit::INT_VECT doneNbrs;
     const auto &enbrs = d_eatoms[dai].neighs;
-    while (nbrIdx != endNbrs) {
-      if (std::find(enbrs.begin(), enbrs.end(), static_cast<int>(*nbrIdx)) ==
+    for (auto nbridx : boost::make_iterator_range(
```

Comment:
I don't think we really expose the boost graph implementation (`boost::make_iterator_range` isn't connected to boost graph and is really just being used because C++14 doesn't have anything to help), but we totally should make the API range friendly.